content/_code-samples/use-tickets/js/use-tickets-multisig.js

```diff
@@ -0,0 +1,129 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // List which Tickets are outstanding against one’s own account and use Tickets to collect signatures for multisign transactions
+  // https://xrpl.org/use-tickets.html
+  // https://xrpl.org/signerlistset.html#signerlistset
+  // https://xrpl.org/multi-signing.html#multi-signing
+
+  async function main() {
+    // Connect to a testnet node
+    console.log("Connecting to Testnet...")
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233')
+    await client.connect()
+  
+    // Get account credentials from the Testnet Faucet
+    console.log("Requesting account credentials from the Testnet faucet, this may take awhile...")
+    const { wallet: main_wallet } = await client.fundWallet()
+
+    // Signer keys don't need to be funded on the ledger, it only needs to be cryptographically valid
+    // Thus, we could generate keys and set them as signers without the need to fund their accounts
+    // But we'll still fund them for testing purposes...
+    const { wallet: wallet_1 } = await client.fundWallet()
+    const { wallet: wallet_2 } = await client.fundWallet()
+    const { wallet: wallet_3 } = await client.fundWallet()
+
+    console.log(" Main Account: ", main_wallet.address)
+    console.log("         Seed: ", main_wallet.seed)
+
+    console.log("\n Signer 1: ", wallet_1.address)
+    console.log(" Signer 2: ", wallet_2.address)
+    console.log(" Signer 3: ", wallet_3.address)
+
+    // Send SignerListSet transaction
+    // Since each signer is given a signer weight of 1 and there are 3 signers, the maximum quorom would be 3
+    // SignerQuorom is a target number for the signer weights
+    // A multisig from this list is valid only if the sum weights of the signatures provided is greater than or equal to the SignerQuorom
+    const signerLiSetSignerList_tx = {
+        TransactionType: "SignerListSet",
+        Account: main_wallet.classicAddress,
```

Comment:
The variable naming convention isn't consistent. For example, there's camelCase mixed with snake_case: `signerLiSetSignerList_tx` and `main_wallet`. Can you refactor the names to be JS camelCase convention?

Reply:
I think that was a typo, sorry, I'll fix it